agent/acs/handler/heartbeat_handler_test.go

```diff
@@ -0,0 +1,95 @@
+// +build unit
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package handler
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/aws/amazon-ecs-agent/agent/acs/model/ecsacs"
+	mock_wsclient "github.com/aws/amazon-ecs-agent/agent/wsclient/mock"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/golang/mock/gomock"
+)
+
+const (
+	heartbeatMessageId = "heartbeatMessageId"
+	heartbeatHealthy   = true
+)
+
+func TestAckHeartbeatMessage(t *testing.T) {
+	heartbeatReceived := &ecsacs.HeartbeatMessage{
+		MessageId: aws.String(heartbeatMessageId),
+		Healthy:   aws.Bool(heartbeatHealthy),
```

Comment:
the Healthy field is just ignored correct? could you add a unit test with Healthy==false just to verify the same behavior?

Reply:
Sure, will add in next rev